user-service/Dockerfile

```diff
@@ -8,4 +8,4 @@ WORKDIR /app
 
 RUN go build -o main . 
 
-CMD ["/app/main"]
+CMD ["/app/main"]
```

Comment:
You may remove this dockerfile if u deem it unnecessary.

Reply:
I think it'll be useful for production.